org.kde.kbreakout.json

```
@@ -0,0 +1,38 @@
+{
+    "id": "org.kde.kbreakout",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kbreakout",
+    "rename-icon": "kbreakout",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
```

```suggestion
        "--socket=fallback-x11",
```

if it supports wayland